### PR TITLE
Better name for app state

### PR DESCRIPTION
**Pull request by lipis**
Example: `Untitled-2020-04-07-1509` and as a filename it will be: 

`Untitled-2020-04-07-1509.excalidraw`

It's also translatable now.

### CODE REVIEW

File: src/locales/en.json [hunk 1]
```diff
@@ -46,7 +46,8 @@
     "actions": "Actions",
     "language": "Language",
     "createRoom": "Share a live-collaboration session",
-    "duplicateSelection": "Duplicate selected elements"
+    "duplicateSelection": "Duplicate selected elements",
+    "untitled": "Untitled"
```
Comment (lipis) [inline src/locales/en.json:50]: @dwelle Are you OK sorting that file now? It will be easier to navigate.

**Comment by lipis** [comment]
@dwelle Can you update the README on how exactly we can update the snapshots?

**Comment by dwelle** [comment]
Prev PR wasn't due to snapshots. This one is. `yarn test -u` doesn't work as @Fausto95  suggested (I reckon that works for bare `jest`). So run all and the `u`. I'll update CONTRIBUTING.

**Comment by lipis** [comment]
Green!

**Pull request merged**